onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -158,6 +158,9 @@ namespace Dml
         std::shared_ptr<onnxruntime::IAllocator> GetCpuInputAllocator();
         std::shared_ptr<onnxruntime::IAllocator> GetCpuOutputAllocator();
 
+        std::shared_ptr<const winrt::Windows::AI::MachineLearning::implementation::InternalRegistrationInfoMap> 
+        GetInternalRegInfoMap() const;
```

Comment:
>Reg [](start = 19, length = 3)

`Registration` preferred.

Reply:
(though I see `m_internalRegInfoMap` is already abbreviate..)

---
In reply to: [369979610](https://github.com/microsoft/onnxruntime/pull/2896#discussion_r369979610) [](ancestors = 369979610)